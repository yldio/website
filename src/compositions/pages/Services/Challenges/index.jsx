/* eslint-disable global-require */

import React from 'react';
import styled from 'styled-components';
import remcalc from 'remcalc';

import { H2, Copy } from 'components/Typography';
import Container from 'components/Container';

const Wrapper = styled.section`
  background: #190533;
  padding: ${remcalc(110)} 0 ${remcalc(90)} 0;
`;

const List = styled.ul`
  margin-top: ${remcalc(40)};
  display: flex;
  flex-wrap: wrap;

  li {
    width: 33%;
    box-sizing: border-box;

    &:not(:last-child) {
      padding-right: 40px;
    }
  }
`;

const Bar = styled.div`
  background: #5e5cb2;
  width: ${remcalc(100)};
  height: ${remcalc(4)};
  margin-top: ${remcalc(-15)};
  margin-bottom: ${remcalc(40)};
`;

const Challenges = () => (
  <Wrapper data-selector="services:challenges">
    <Container>
      <H2 white>Common client challenges</H2>
      <Bar />
      <Copy light>
        We help you future proof your business through a new style of
        consulting, ground up.
      </Copy>
      <List>
        <li>
          <img
            src={require('../../../../assets/pair.svg')}
            alt="Customer experience"
            width="49px"
            height="41px"
          />
          <Copy white uppercase>
            Customer experience
          </Copy>
          <Copy light>
            Optimise people, processes, user experience, design and technology
            <br />
            <br />
            Provide a seamless customer experience across different channels
          </Copy>
        </li>
        <li>
          <img
            src={require('../../../../assets/teams.svg')}
            alt="Disruptive technology"
            width="55px"
            height="49px"
          />
          <Copy white uppercase>
            Disruptive technology
          </Copy>
          <Copy light>
            Align to your business goals but stay lean and innovate
            <br />
            <br />
            Stay ahead of the market
            <br />
            <br />
            Compete with start ups
          </Copy>
        </li>
        <li>
          <img
            src={require('../../../../assets/blog.svg')}
            alt="Skills Gap"
            width="49px"
            height="41px"
          />
          <Copy white uppercase>
            Skills Gap
          </Copy>
          <Copy light>
            Bridge the skills gap and attract the right technology professionals
            <br />
            <br />
            Achieve your business goals
            <br />
            <br />
            Make digital, technological or cultural change happen.
          </Copy>
        </li>
      </List>
    </Container>
  </Wrapper>
);

export default Challenges;
