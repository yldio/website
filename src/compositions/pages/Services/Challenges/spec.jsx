import React from 'react';
import { shallow } from 'enzyme';
import Challenges from './index';

test('compositions/pages/Services/Challenges snapshot', () => {
  const wrapper = shallow(<Challenges />);

  expect(wrapper).toMatchSnapshot();
});
