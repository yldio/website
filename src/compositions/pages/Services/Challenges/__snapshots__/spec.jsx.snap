// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/pages/Services/Challenges snapshot 1`] = `
<Challenges__Wrapper
  data-selector="services:challenges"
>
  <Container>
    <Typography__H2
      white={true}
    >
      Common client challenges
    </Typography__H2>
    <Challenges__Bar />
    <Typography__base
      light={true}
    >
      We help you future proof your business through a new style of consulting, ground up.
    </Typography__base>
    <Challenges__List>
      <li>
        <img
          alt="Customer experience"
          height="41px"
          src="/src/assets/pair.svg"
          width="49px"
        />
        <Typography__base
          uppercase={true}
          white={true}
        >
          Customer experience
        </Typography__base>
        <Typography__base
          light={true}
        >
          Optimise people, processes, user experience, design and technology
          <br />
          <br />
          Provide a seamless customer experience across different channels
        </Typography__base>
      </li>
      <li>
        <img
          alt="Disruptive technology"
          height="49px"
          src="/src/assets/teams.svg"
          width="55px"
        />
        <Typography__base
          uppercase={true}
          white={true}
        >
          Disruptive technology
        </Typography__base>
        <Typography__base
          light={true}
        >
          Align to your business goals but stay lean and innovate
          <br />
          <br />
          Stay ahead of the market
          <br />
          <br />
          Compete with start ups
        </Typography__base>
      </li>
      <li>
        <img
          alt="Skills Gap"
          height="41px"
          src="/src/assets/blog.svg"
          width="49px"
        />
        <Typography__base
          uppercase={true}
          white={true}
        >
          Skills Gap
        </Typography__base>
        <Typography__base
          light={true}
        >
          Bridge the skills gap and attract the right technology professionals
          <br />
          <br />
          Achieve your business goals
          <br />
          <br />
          Make digital, technological or cultural change happen.
        </Typography__base>
      </li>
    </Challenges__List>
  </Container>
</Challenges__Wrapper>
`;
